src/firebase.js

```diff
@@ -6,13 +6,13 @@ import "firebase/compat/firestore";
 import "firebase/compat/database";
 
 const firebaseConfig = {
-  apiKey: process.env.REACT_APP_API_KEY,
```

Comment:
Why did you change this file?

Reply:
@BamaCharanChhandogi I did this because it is not working locally in my system. It is having some uncaught runtime error.

Reply:
yeah same problem, i created a seperate firebase config, still not working

Reply:
I have added a .env.example file. Tell me if any issues are coming up or not!

Reply:
@BamaCharanChhandogi Yeah it is working fine